lib/amplitude.dart

```diff
@@ -147,7 +147,7 @@ class Amplitude extends _Amplitude {
       event.mergeEventOptions(options);
     }
 
-    return await _channel.invokeMethod("track", jsonEncode(event.toMap()));
```

Comment:
Why does this change? Was the old version calling the wrong method or did the interface change in the new Android SDK?

Reply:
This is to fix the error of my last commit. Note this PR is compared with branch `4.x` not `main`. The interface `setGroup()` remains unchanged.